### ISSUE TITLE
[BUG] Error 403 after installation

### ISSUE THREAD

**Issue by Armellys** [certsocietegenerale/swordphish-awareness#11]
Hi,

I have an issue once the installation is done.
I can connect on the "SWORPDHISH_URL"/login or on "SWORPDHISH_URL"/admin.
But when I try to log in, I have a 403 error and I don't know from where this could come from.
I am working with a vm Ubuntu server 16.04 on virtualbox.
Here is the log access I have : 
192.168.98.72 - - [22/Mar/2019:10:51:26 +0100] "GET /login HTTP/1.1" 200 978 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:65.0) Gecko/20100101 Firefox/65.0"
192.168.98.72 - - [22/Mar/2019:10:51:26 +0100] "GET /favicon.ico HTTP/1.1" 404 2079 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:65.0) Gecko/20100101 Firefox/65.0"
192.168.98.72 - - [22/Mar/2019:10:51:36 +0100] "POST /login HTTP/1.1" 302 0 "https://192.168.98.141/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:65.0) Gecko/20100101 Firefox/65.0"
192.168.98.72 - - [22/Mar/2019:10:51:36 +0100] "GET / HTTP/1.1" 403 0 "https://192.168.98.141/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:65.0) Gecko/20100101 Firefox/65.0"
192.168.98.72 - - [22/Mar/2019:10:51:36 +0100] "GET /favicon.ico HTTP/1.1" 404 2079 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:65.0) Gecko/20100101 Firefox/65.0"

**Comment by Heat-Miser** [comment]
Hello,

can you enable debug in settings file if not already done ?  Is there any specific message related to ALLOWED_HOSTS ?

Thx

**Comment by Armellys** [comment]
I find the solution.
It was the HOSTING_DOMAIN which was not correct, It was set on localhost instead of the host used to access the interface.

**Comment by Heat-Miser** [comment]
Great !

I close this issue so.
